### PR TITLE
Deal cards initial commit to dev

### PR DESCRIPTION
**Pull request by dpaschal-lf**
still a lot of work to do including:

- fix animation
- move card appending and other card / deck operations to their appropriate object
- fix that damn player area blue box so that it doesn't float above everything else

### CODE REVIEW

File: includes/js/rootbrawl.js [hunk 1]
```diff
@@ -88,6 +88,7 @@ function RootBrawl(gameAreaElement){
 	this.playerCreated = function(playerDomElement){
 		this.gameArea.append(playerDomElement);
 		this.player.createDeck(deckStats);
+		this.player.createHand();
 		//TODO: get deckStats from a parameter or load through a property
```
Comment (MotM-MrRager) [inline includes/js/rootbrawl.js]: As far as deckStats go, were there any particular stats you were looking for, or were you looking for all the stats?

Comment (MotM-MrRager) [inline includes/js/rootbrawl.js]: As far as deckStats go, were there any particular stats you were looking for, or were you looking for all the stats?

File: includes/js/rootbrawl.js [hunk 2]
```diff
@@ -115,7 +116,7 @@ function Player(){
 	}
 	this.cardHolders = {
 		deck: null,
-		hand: [],
+		hand: null,
```
Comment (MotM-MrRager) [inline includes/js/rootbrawl.js]: **Priority of Question:** _LOW_
Any particular reason hand's assignment shifted from an empty array to null?

File: includes/js/rootbrawl.js (hunk 1 above)
Comment (MotM-MrRager) [inline includes/js/rootbrawl.js]: Can you expand on what you were looking for here? I wrote a draft of a function that goes through the keys and values for the cards and console logs them (for now), so I was wondering what you would want to use them for to better tailor the function's output/behavior

File: includes/js/rootbrawl.js (hunk 2 above)
Comment (dpaschal-lf) [inline includes/js/rootbrawl.js]: because I don't want to store an array of cards, I want to store the reference to the deck @MotM-MrRager

**Pull request merged**